lib/configgin.rb

```diff
@@ -66,6 +87,51 @@ def render_job_templates(jobs, job_configs)
     end
   end
 
+  # Some pods might have depended onthe properties exported by this pod; locate
+  # them and cause them to restart as appropriate.
+  def restart_affected_pods(job_configs, job_digests)
```

Comment:
There is quite a bit of logic in this one. How about a unit test for it?

Reply:
Refactored and added in 26897e5e22937ebd5e0e1cb5ca5f15c4fffad2a3